content/docs/2.0/operate/cluster.md

```diff
@@ -36,11 +36,11 @@ Here is an overview of the required ports that need to be accessible for KEDA to
 
 ## High Availability
 
-KEDA does not provide support for high-availability due to upstream limitations.
+KEDA does not provide full support for high-availability due to upstream limitations.
 
-Here is an overview of all KEDA deployments and the supported replicas:
+Here is an overview of all KEDA deployments and the HA notes:
 
-| Deployment     | Support Replicas        | Reasoning                     |
-|----------------|-------------------------|-------------------------------|
-| Operator       | 1                       |                               |
-| Metrics Server | 1                       | Limitation in [k8s custom metrics server](https://github.com/kubernetes-sigs/custom-metrics-apiserver/issues/70) |
\ No newline at end of file
+| Deployment     | Note                                                                                                                                                                                                                   |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Metrics Server | You can run multiple replicas of our metrics sever, and it is recommended to add the `--enable-aggregator-routing=true` CLI flag to the kube-apiserver so that requests sent to our metrics servers are load balanced. |
+| Operator       | While you can run multiple replicas of our operator, only one operator instance will be active. The rest will be standing by, which may reduce downtime during a failure. Multiple replicas will not improve the performance of KEDA, it could only reduce a downtime during a failover.<br /><br />This is only supported as of KEDA v2.6 if you are using our Helm chart.|
```

Comment:
For clarity, maybe this column should be named `Default Replicas` rather than `Support Replicas`? Because it's just the default value in chart, and KEDA can actually support more replicas.

Reply:
It's not because you can do it that we actively support it though.

For example, if you decide to start running 5 instances of the operator and we introduce memory caching it will work but will give edge scenarios that we don't support (yet) unless documented otherwise.

That's the idea why we have/had this

Reply:
Got it, changed.